client/src/App.js

```diff
@@ -1,29 +1,34 @@
 import React, { Component } from 'react';
-
+import { Route } from 'react-router-dom';
 import SavedList from './Movies/SavedList';
 import MovieList from './Movies/MovieList';
-import Movie from './Movies/Movie';
+import Movie from './Movies/Movie'
 
 export default class App extends Component {
-  constructor() {
-    super();
+  constructor(){
+    super()
     this.state = {
       savedList: []
-    };
+    }
   }
 
-  addToSavedList = movie => {
+  addToSavedList = (movie) => {
+    console.log(this.state.savedList)
     const savedList = this.state.savedList;
     savedList.push(movie);
-    this.setState({ savedList });
-  };
+    this.setState({savedList});
+  }
```

Comment:
Challenge: Right now `addToSavedList` is adding movie to the savedList even though the movie is already saved. Can you think of away to refactor this method so that it will only add a movie to the savedList if the movie has not been added before?